apollo/organisms/__init__.py

```diff
@@ -12,7 +12,7 @@ class OrganismsClient(Client):
 
     @raise_error_decorator
     def add_organism(self, common_name, directory, blatdb=None, genus=None,
-                     species=None, public=False, metadata=None):
+                     species=None, public=False, metadata=None, return_all=None):
```

Comment:
This should be False, not None

Reply:
This one was merged, but I can make another PR for that

Reply:
Oh it was replaced by `suppress_output=False` in 4.2.7

Reply:
ahh ok. cool, just caught up on a too-old pr then.

Reply:
no problem :)